lib/IHP/static/livereload.js

```diff
@@ -27,6 +27,10 @@ function refresh() {
         .then(html => {
             var parser = new DOMParser();
             var dom = parser.parseFromString(html, 'text/html');
+
+            // https://github.com/digitallyinduced/ihp/issues/998
+            morphdom(document.head, dom.head)
+
```

Comment:
I wonder if this will help us with our problem of Elm not interacting well with the livereload.